node/src/accountant/scanners/payable_scanner/utils.rs

```diff
@@ -0,0 +1,565 @@
+        let ideal_template_middle: i128 = gwei_to_wei(
+            (payment_thresholds.debt_threshold_gwei
+                - payment_thresholds.permanent_debt_allowed_gwei)
+                / 2
+                + payment_thresholds.permanent_debt_allowed_gwei,
+        );
+        let ideal_template_lower: i128 =
+            gwei_to_wei(payment_thresholds.permanent_debt_allowed_gwei);
+        assert!(
+            higher_corner_point <= ideal_template_higher + allowed_imprecision
+                && ideal_template_higher - allowed_imprecision <= higher_corner_point,
+            "ideal: {}, real: {}",
+            ideal_template_higher,
+            higher_corner_point
+        );
+        assert!(
+            middle_point <= ideal_template_middle + allowed_imprecision
+                && ideal_template_middle - allowed_imprecision <= middle_point,
+            "ideal: {}, real: {}",
+            ideal_template_middle,
+            middle_point
+        );
+        assert!(
+            lower_corner_point <= ideal_template_lower + allowed_imprecision
+                && ideal_template_lower - allowed_imprecision <= lower_corner_point,
+            "ideal: {}, real: {}",
+            ideal_template_lower,
+            lower_corner_point
+        )
+    }
+
+    #[test]
+    fn is_innocent_age_works_for_age_smaller_than_innocent_age() {
+        let payable_age = 999;
+
+        let result = PayableThresholdsGaugeReal::default().is_innocent_age(payable_age, 1000);
+
+        assert_eq!(result, true)
+    }
+
+    #[test]
+    fn is_innocent_age_works_for_age_equal_to_innocent_age() {
+        let payable_age = 1000;
+
+        let result = PayableThresholdsGaugeReal::default().is_innocent_age(payable_age, 1000);
+
+        assert_eq!(result, true)
+    }
+
+    #[test]
+    fn is_innocent_age_works_for_excessive_age() {
+        let payable_age = 1001;
+
+        let result = PayableThresholdsGaugeReal::default().is_innocent_age(payable_age, 1000);
+
+        assert_eq!(result, false)
+    }
+
+    #[test]
+    fn is_innocent_balance_works_for_balance_smaller_than_innocent_balance() {
+        let payable_balance = 999;
+
+        let result =
+            PayableThresholdsGaugeReal::default().is_innocent_balance(payable_balance, 1000);
+
+        assert_eq!(result, true)
+    }
+
+    #[test]
+    fn is_innocent_balance_works_for_balance_equal_to_innocent_balance() {
+        let payable_balance = 1000;
+
+        let result =
+            PayableThresholdsGaugeReal::default().is_innocent_balance(payable_balance, 1000);
+
+        assert_eq!(result, true)
+    }
+
+    #[test]
+    fn is_innocent_balance_works_for_excessive_balance() {
+        let payable_balance = 1001;
+
+        let result =
+            PayableThresholdsGaugeReal::default().is_innocent_balance(payable_balance, 1000);
+
+        assert_eq!(result, false)
+    }
+
+    #[test]
+    fn requires_payments_retry_says_yes() {
```

Comment:
Please delete this test. It is originally addressed to the PendingPayableScanner and so it should not be in this file. 
I have repaired these tests in my card, you can have no worries about them.